pkgs/applications/networking/instant-messengers/wire-desktop/default.nix

```diff
@@ -25,14 +25,20 @@ let
 
   pname = "wire-desktop";
 
-  version = {
-    x86_64-darwin = "3.31.4556";
-    x86_64-linux = "3.31.3060";
+  version = let
+    x86_64-darwin = "3.32.4589";
+  in {
+    inherit x86_64-darwin;
+    aarch64-darwin = x86_64-darwin;
```

Comment:
does just reusing the `x86_64-darwin` binaries and packing it up, pretending it's `aarch64-darwin` work here?

Reply:
They only have a single bundle for macOS. It's customary on macOS to do "universal" binaries that include all the architectures currently supported officially by Apple.

Reply:
Can you add a comment above that line mentioning it?

Reply:
If you think it's really necessary but this is a very common thing on macOS. The only reason I'm keeping the architecture specific platform keys rather than just having a single source is the `.${system}` setup.